compose/build.gradle.kts

```diff
@@ -0,0 +1,125 @@
+            export(libs.moko.resources)
+        }
+    }
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                implementation(project(mapOf("path" to ":domain")))
+                implementation(project(mapOf("path" to ":shared")))
+                // Put your multiplatform dependencies here
+
+                // Jetpack Compose
+                implementation(compose.runtime)
+                implementation(compose.ui)
+                implementation(compose.foundation)
+                implementation(compose.animation)
+                implementation(compose.material3)
+                @OptIn(org.jetbrains.compose.ExperimentalComposeLibrary::class)
+                implementation(compose.components.resources)
+
+                // PreCompose - https://github.com/Tlaster/PreCompose
+                api(libs.precompose)
+                api(libs.precompose.viewmodel)
+                api(libs.precompose.koin)
+                api(libs.moko.resources)
+                api(libs.moko.resources.compose)
+
+                // KTOR Networking and Serialization
+                implementation(libs.ktor.client.core)
+                implementation(libs.ktor.client.content.negotiation)
+                implementation(libs.ktor.serialization.kotlinx.json)
+                implementation(libs.koin.core)
+
+                // Logger
+                implementation(libs.kermit.logger)
+            }
+        }
+        val androidMain by getting {
+            dependsOn(commonMain)
+            dependencies {
+                implementation(libs.kotlinx.coroutines.core)
+                implementation(libs.ktor.client.android)
+                implementation(libs.koin.android)
+                implementation(libs.koin.androidx.compose)
+
+                // Utility
+                implementation(libs.google.maps)
+                implementation(libs.google.maps.utils)
+                implementation(libs.google.places)
+                implementation(libs.play.services.maps)
+
+                // Preview Utils need to be implemented in platform code as they use platform renderers
+                implementation(compose.preview)
+                implementation(compose.uiTooling)
+            }
+        }
+        val iosMain by getting {
+            dependsOn(commonMain)
+            dependencies {
+                implementation(libs.ktor.client.darwin)
+            }
+        }
+        val commonTest by getting {
+            dependsOn(commonMain)
+            dependencies {
+                implementation(libs.koin.test)
+                implementation(libs.moko.resources.test)
+            }
+        }
+    }
+}
+
+multiplatformResources {
+    multiplatformResourcesPackage = "com.br.kmmdemo.resources"
+    multiplatformResourcesClassName = "SharedRes"
+}
+
+android {
+    namespace = "com.br.kmmdemo"
+    compileSdk = 34
+    defaultConfig {
+        minSdk = 30
```

Comment:
There's a cool way to specify these in the library catalog (toml file). look at some of the KMP launchpad libs for the pattern